main.md

```diff
@@ -1,9 +1,26 @@
-## GIT CHEATSHEET
+# GIT CHEATSHEET
 
-### Common Commands
+## Initializing Git
+Those commands are related to the inizialitazion process for git repositories on your local machine.
+
+`$ git init`
+- It will create a new git repository in the current directory.
+
+`$ git init --bare`
+- It will create a new bare repository.
+
+A bare repository is a bit different from a regular one, it doesn't .git folder, history is stored in the project root, also if you try to `git clone --bare` from a remote repository (e.g. github) you will lost track of it's origin since usually bare repositories are supposed to be served at the users as remote endpoints.
+
+## Cloning Repositories
 `$ git clone <link to repo>`
 - Clones repository to local machine.
 
+### Shallow cloning
+$ git clone <link> --depth=1
```

Comment:
fix this, add it in ``

Reply:
> @Nhoya Since all your commits are similar...can u squash those?

All of them?

Reply:
Chuck it..

Reply:
Oh well, the rebase wasn't working as intended... probably ZSH is not working in my favour, let me just reopen a PR